wedlocks/src/templates/confirmEvent.js

```diff
@@ -1,9 +1,17 @@
+const ethWaterlooMessage = `
+
+After you check in to ETHWaterloo, we'll refund your stake in DAI by the end of the weekend. You'll be able to access those funds once you eject your Unlock account into a crypto wallet. We'll provide more details on that process soon! Note that this event ticket is non-transferable.`
+
 export default {
-  subject: () => 'You are attending an event!',
+  subject: params => `You've got your ticket for ${params.eventName}!`,
   text: params =>
     `Hello,
 
-This is just a reminder that you are attending the event ${params.eventName} on ${params.eventDate}!
+This is just a reminder that you are attending the event ${
+      params.eventName
+    } on ${params.eventDate}!${
+      params.eventName === 'ETHWaterloo' ? ethWaterlooMessage : ''
```

Comment:
I'm dispatching on `eventName` here, but it might be wiser to do it by matching the lock address in the `ticketLink` field

Reply:
Who currently owns the Lock? Do we have a risk of it changing? The lock Address should at least be consistent though

Reply:
Julien created the lock, not sure if he transferred ownership to ETHWaterloo yet. In any event, the address should remain the same always.